README.md

```diff
@@ -223,15 +223,16 @@ For examples:
 > **NOTE:** In the following conventional paths for examples, `<data source name>` and `<resource name>` include the provider prefix as well, but the provider prefix is **NOT** included in`<function name>`.
 > For example, the data source [`caller_identity`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/caller_identity) in the `aws` provider would have an "example" conventional path of: `examples/data-sources/aws_caller_identity/data-source.tf`
 
-| Path                                                                      | Description                       |
-|---------------------------------------------------------------------------|-----------------------------------|
-| `examples/`                                                               | Root of examples                  |
-| `examples/provider/provider.tf`                                           | Provider example config           |
-| `examples/data-sources/<data source name>/data-source.tf`                 | Data source example config        |
-| `examples/ephemeral-resources/<ephemeral resource>/ephemeral-resource.tf` | Ephemeral resource example config |
-| `examples/functions/<function name>/function.tf`                          | Function example config           |
-| `examples/resources/<resource name>/resource.tf`                          | Resource example config           |
-| `examples/resources/<resource name>/import.sh`                            | Resource example import command   |
+| Path                                                                      | Description                          |
+|---------------------------------------------------------------------------|--------------------------------------|
+| `examples/`                                                               | Root of examples                     |
+| `examples/provider/provider.tf`                                           | Provider example config              |
+| `examples/data-sources/<data source name>/data-source.tf`                 | Data source example config           |
+| `examples/ephemeral-resources/<ephemeral resource>/ephemeral-resource.tf` | Ephemeral resource example config    |
+| `examples/functions/<function name>/function.tf`                          | Function example config              |
+| `examples/resources/<resource name>/resource.tf`                          | Resource example config              |
+| `examples/resources/<resource name>/import.sh`                            | Resource example import command      |
+| `examples/resources/<resource name>/import-id.tf`                         | Resource example import by id config |
```

Comment:
> `import-id.tf`

Open to other suggestions for naming on this, originally I had `import.tf` but I think it's a good idea to add `id` in some way because I think we'll want to add `identity` as a 2nd type of import configuration example. So with this current naming, we'd introduce another at:
-   `examples/resources/<resource name>/import-identity.tf`

--------

Other idea I was thinking of doing was `import-by-id.tf` and `import-by-identity.tf`, but was split between the two options 😆

Reply:
Yeah I can see that 🤔, what do you think about:
- `import-by-string-id.tf`
- `import-by-identity.tf`